### PR TITLE
Workaround problems with cmake and xz on manylinux2014

### PR DESCRIPTION
**Pull request by radarhere**
Helps https://github.com/python-pillow/pillow-wheels/pull/147 by working around the problems with manylinux2014.

However, feel free to also just tell me that these should be incorporated into multibuild. I imagine you're in a good place to make that call.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -164,7 +164,7 @@ jobs:
       os: linux
       env:
         - MB_PYTHON_VERSION=pypy3.6-7.3.1
-        - MB_ML_VER=2010
+        - MB_ML_VER=2014
```
Comment (mattip) [inline .travis.yml:167]: This is on purpose: PyPy needs manylinux2010 but doesn't require manylinux2014. Reverted

File: config.sh [hunk 2]
```diff
@@ -24,6 +24,16 @@ function pre_build {
...
+    elif [ $MB_ML_VER -eq 2014 ]; then
+        yum install -y cmake
+    
+        function get_cmake {
+            echo cmake
+        }
+
+        function build_xz {
+            yum install -y xz-devel
+        }
```
Comment (mattip) [inline config.sh:36]: Thanks!

**Comment by mattip** [comment]
I am not sure what the policy should be around outdated multibuild components. Maybe we need a multibuild2010 and multibuild2014 with different library versions.

File: .travis.yml (hunk 1 above)
Comment (radarhere) [reply .travis.yml:167]: Just to have it written down, why the preference for the earlier one?

Comment (mattip) [reply .travis.yml:167]: The older version should make the wheels compatible with marginally more versions of linux.

**Pull request merged**